src/Microsoft.AspNet.Security.OpenIDConnect/NonceCache.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Security.OpenIdConnect
+{
+    public interface INonceCache
```

Comment:
Ahah thanks, I was indeed referring to `Microsoft.IdentityModel.Protocol.Extensions` but I couldn't remember the exact name :smile:

`System.IdentityModel.Jwt` could also be an option, but this interface is not directly specific to JWT tokens.